target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.c

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-2.0-only
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
SPDX identifiers with `//` are correct in a source file.
See: https://www.kernel.org/doc/html/v4.18/process/license-rules.html#license-identifier-syntax